tests/scripts/populate_indices_for_cloud_tests.py

```diff
@@ -0,0 +1,142 @@
+            [{}],
+        "cool-index" + '-' + test_uniqueness_id:
+            [{}],
+        "first-index" + '-' + test_uniqueness_id:
+            [{}],
+        "weight-index" + '-' + test_uniqueness_id:
+            [{}],
+        "mmodal-index" + '-' + test_uniqueness_id:
+            [{"treat_urls_and_pointers_as_images": True, "model": "ViT-B/32"}],
+        generic_test_index_name + '-' + test_uniqueness_id:
+            [{},
+             {"settings_dict": {
+                 "index_defaults": {
+                     "ann_parameters": {
+                         "parameters": {
+                             "m": 24
+                         }
+                     }
+                 }
+             }
+              },
+             {"treat_urls_and_pointers_as_images": True,
+              "model": "ViT-B/32",
+              },
+             {"model": "ViT-B/32"
+              },
+             {"settings_dict":
+                 {
+                     'index_defaults': {
+                         'treat_urls_and_pointers_as_images': False,
+                         'model': 'test-model',
+                         'model_properties': {
+                             'name': 'sentence-transformers/multi-qa-MiniLM-L6-cos-v1',
+                             'dimensions': 384,
+                             'tokens': 128,
+                             'type': 'sbert'
+                         },
+                         'normalize_embeddings': True,
+                     }
+                 }
+              },
+             {"treat_urls_and_pointers_as_images": True,  # allows us to find an image file and index it
+              "model": "ViT-B/16",
+              "image_preprocessing_method": None
+              },
+             {"treat_urls_and_pointers_as_images": True,  # allows us to find an image file and index it
+              "model": "ViT-B/16",
+              "image_preprocessing_method": "simple"
+              },
+             {"sentences_per_chunk": int(1e3),
+              "sentence_overlap": 0
+              },
+             {"sentences_per_chunk": 2,
+              "sentence_overlap": 0
+              },
+             {"sentences_per_chunk": 2,
+              "sentence_overlap": 1
+              },
+             {"settings_dict":
+                 {
+                     'index_defaults': {
+                         'model': "ViT-B/16",
+                         'treat_urls_and_pointers_as_images': True
+                     }
+                 }
+              },
+             ]
+    }
+
+    marqo_settings = {
+        "url": os.environ.get("MARQO_URL", 'http://localhost:8882'),
+    }
+    api_key = os.environ.get("MARQO_API_KEY", None)
+    if api_key:
+        marqo_settings["api_key"] = api_key
+
+    mq = marqo.Client(**marqo_settings)
+
+    indexes_to_create = []
+
+    for index_name, configs_list in index_name_to_config_mappings.items():
+        for config in configs_list:
```

Comment:
calling this config is confusing (I thought it was referring to an instance of the Config class). How about calling this index_configs, for example?